synthea_omop_etl/models/omop/schema.yml

```diff
@@ -1,5 +1,12 @@
 version: 2
 
+sources:
```

Comment:
can you please move the sources block into the base/schema.yml instead? it will be cleaner to store this config alongside the base models.